lib/linter/linter.js

```diff
@@ -510,6 +510,7 @@ function normalizeVerifyOptions(providedOptions, config) {
 
     return {
         filename: normalizeFilename(providedOptions.filename || "<input>"),
+        physicalFilename: normalizeFilename(providedOptions.physicalFilename || "<input>"),
```

Comment:
Per other comments, we don’t want physicalFilename to be accepted as an option to be passed linter.verify(), so I think this can be removed.

Reply:
I think then we will need `getPhysicalFilename: () => physicalFilename || filename,`

Reply:
As `physicalFilename` will only be defined in when we have `options.preprocess` or `options.postprocess`